src/taoensso/sente.cljx

```diff
@@ -81,6 +81,16 @@
   (:require-macros
    [cljs.core.async.macros :as asyncm :refer (go go-loop)]))
 
+#+cljs
+(def is-node? (nil? enc/js-?win))
```

Comment:
Not sure if this is the best detection for `is-node?`, perhaps https://github.com/iliakan/detect-node/blob/master/index.js or another method would be safer?